Makefile

```diff
@@ -15,6 +15,7 @@ NANOSERVER_VERSION := $(shell head -n 1 build/images/deps/nanoserver-version)
 WIN_BUILD_TAG      := $(shell echo $(GO_VERSION) $(CNI_BINARIES_VERSION) $(NANOSERVER_VERSION)|md5sum|head -c 10)
 GIT_HOOKS := $(shell find hack/git_client_side_hooks -type f -print)
 DOCKER_NETWORK     ?= default
+TRIVY_TARGET_IMAGE :=
```

Comment:
@liu4480 should it be `?=` instead of `:=`?

I tested the following and it failed:
```bash
abas-a01:antrea abas$ export TRIVY_TARGET_IMAGE=antrea/antrea-ubuntu
abas-a01:antrea abas$ make trivy-scan
Makefile:58: *** TRIVY_TARGET_IMAGE is undefined.  Stop.
```

Reply:
thanks, updated